tests/neg-custom-args/fatal-warnings/i4674.scala

```diff
@@ -0,0 +1,9 @@
+object Testi4674 {
+  def T(x: String) = {
+    x.foreach {
+      case 's' => println("s")
+      case c: Char => println(c) // error
```

Comment:
Can you write `// error: type test always succeed` instead. Otherwise we don't what is the error without running the test